### PR TITLE
Add EverythingLM Dataset

### PR DESCRIPTION
**Pull request by manandey**
Hi @Muennighoff,

I have added the EverythingLM dataset. It would be great if you would have a look and it and let me know if any changes are required.

Thanks,
Manan

### CODE REVIEW

**Comment by Muennighoff** [comment]
> Hi @Muennighoff,
> 
> I have added the EverythingLM dataset. It would be great if you would have a look and it and let me know if any changes are required.
> 
> Thanks, Manan

Great work. Did you run the tests & did they all pass?

**Comment by manandey** [comment]
Yeah, I ran the tests specific to this dataset and all of them passed.

File: data_summaries/EverythingLM.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                "License URL": "https://huggingface.co/datasets/totally-not-an-llm/EverythingLM-data-V3"
+            }
+        ],
+        "License Notes": "",
+        "License Verified By": "Manan",
+        "Dataset Filter IDs": [
+            "everything_lm"
+        ]
+    }
+}
```
Comment (Muennighoff) [inline data_summaries/EverythingLM.json:46]:
```suggestion
}

```

**Review by Muennighoff — APPROVED** [review]
Great work!

**Comment by shayne-longpre** [comment]
@manandey @Muennighoff looks great! Sorry for the late request, but wondering if we can make a couple minor changes:
* Confirm it passed the tests?
* Could we retroactively add the OpenAI
* It looks like in HF they have a pie chart with a few more tasks we didn't include in the Task Categories. Any chance we could expand our list to make sure its covering all of theres?

**Comment by Muennighoff** [comment]
> @manandey @Muennighoff looks great! Sorry for the late request, but wondering if we can make a couple minor changes:
> 
> * Confirm it passed the tests?
> * Could we retroactively add the OpenAI license/terms to the list of licenses
> * It looks like in HF they have a pie chart with a few more tasks we didn't include in the Task Categories. Any chance we could expand our list to make sure its covering all of theres?

1) I think @manandey checked that one
2) & 3) Sounds good - could you do it @manandey? I thought 2) isn't necessary as the authors of this dataset do not put that license. Some people from industry told me that they are only liable w.r.t. the license that the creators put behind the data (even if that license could be seen as wrong like in this case where OAI license may be more appropriate).

**Comment by shayne-longpre** [comment]
> > @manandey @Muennighoff looks great! Sorry for the late request, but wondering if we can make a couple minor changes:
> > 
> > * Confirm it passed the tests?
> > * Could we retroactively add the OpenAI license/terms to the list of licenses
> > * It looks like in HF they have a pie chart with a few more tasks we didn't include in the Task Categories. Any chance we could expand our list to make sure its covering all of theres?
> 
> 1. I think @manandey checked that one
> 2. & 3) Sounds good - could you do it @manandey? I thought 2) isn't necessary as the authors of this dataset do not put that license. Some people from industry told me that they are only liable w.r.t. the license that the creators put behind the data (even if that license could be seen as wrong like in this case where OAI license may be more appropriate).

Thx! Yeah, I've heard people making that claim as well. We have a toggle in the explorer and filter tool that allows people to ignore that license if they wish to. It's more that we don't want users who are expecting to download data with no implicated terms/licenses to get caught out because we didn't put the label there.

**Comment by manandey** [comment]
> 2. & 3) Sounds good - could you do it @manandey? I thought 2) isn't necessary as the authors of this dataset do not put that license. Some people from industry told me that they are only liable w.r.t. the license that the creators put behind the data (even if that license could be seen as wrong like in this case where OAI license may be more appropriate).

Sure, I will raise a follow-up PR with the changes by today.

**Pull request merged**